core/sockets.py

```diff
@@ -44,7 +44,8 @@
     "SvDummySocket": (0.8, 0.8, 0.8, 0.3),
     "SvSeparatorSocket": (0.0, 0.0, 0.0, 0.0),
     "SvObjectSocket": (0.69, 0.74, 0.73, 1.0),
-    "SvTextSocket": (0.68, 0.85, 0.90, 1)
+    "SvTextSocket": (0.68, 0.85, 0.90, 1),
+    "SvDictionarySocket": (0.40, 0.02, 1.00, 1.00)
```

Comment:
NOOOOOOOOOOO :)  i dislike this colour a lot

Reply:
.links was computed on every lookup.

Reply:
@zeffii @portnov @vicdoval @DolphinDream @nortikin @enzyme69 Please, have a look at examples of color of dictionary sockets, which variant is more pleasant for you?
# Node fotosession
## Lite
**1:** (1,1,1,1)
![2019-12-29_20-47-25](https://user-images.githubusercontent.com/28003269/71559863-71190d80-2a7c-11ea-9721-79aad096fe87.png)

**2:** (0.55, 0.35, 0.75, 1.0)
![2019-12-29_20-46-04](https://user-images.githubusercontent.com/28003269/71559847-416a0580-2a7c-11ea-88b0-ee0277ac0d4b.png)

**3:** (0.85, 0.55, 0.35, 1.0)
![2019-12-29_20-52-26](https://user-images.githubusercontent.com/28003269/71559939-29df4c80-2a7d-11ea-9b00-fa9749ca16d4.png)

## Dark
**1:** (0.85, 0.15, 0.15, 1.0)
![2019-12-29_20-54-55](https://user-images.githubusercontent.com/28003269/71559966-83477b80-2a7d-11ea-89d2-ccf6074d5473.png)

**2:** (0.0, 0.30, 0.80, 1.0)
![2019-12-29_21-00-10](https://user-images.githubusercontent.com/28003269/71560044-3fa14180-2a7e-11ea-9d80-aca62aefcd8d.png)

**3:**  (0.60, 0.00, 0.60, 1.0)
![2019-12-29_21-02-37](https://user-images.githubusercontent.com/28003269/71560069-973fad00-2a7e-11ea-9492-1138a764379c.png)